src/lib/isMobilePhone.js

```diff
@@ -2,7 +2,7 @@ import assertString from './util/assertString';
 
 /* eslint-disable max-len */
 const phones = {
-  'am-AM': /^(\+?374|0)((10|[9|7][0-9])\d{6}$|[2-4]\d{7}$)/,
+  'am-AM': /^(\+?374|0)((10|[5679][0-9])\d{6}$|[2-4]\d{7}$)/,
```

Comment:
```suggestion
  'am-AM': /^(\+?374|0)((33|4[1|3|4]|55|77|88|9[1|3|4|5|6|8|9])\d{6}$)/,
```
According to this [reference from ITU](https://www.itu.int/dms_pub/itu-t/oth/02/02/T020200000A0007PDFE.pdf) (accessed from https://www.itu.int/oth/T020200000A/en) this should be the proper RegExp for mobile numbers.

Can you update the tests accordingly?

Reply:
This validator is only for mobile phone numbers, see 'Non-geographic number for mobile services' on page 12 of above mentioned reference. Phone numbers that are marked as 'fixed telephony services' should not be valid in this validator.

Reply:
ok, got it
I've removed validation for 60xxxxxx numbers